res/css/views/settings/_SpellCheckLanguages.scss

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 New Vector Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
```

Comment:
I guess this should be 2021 Simon Brandner? Or just the foundation, if you don't want personal copyright.